.github/workflows/test.yml

```diff
@@ -8,6 +8,7 @@ jobs:
         gemfile:
           - Gemfile-rails.5.2.x
           - Gemfile-rails.6.0.x
+          - Gemfile-rails.7.0.x
```

Comment:
We need to exclude the 7.0 Gemfile from Ruby 2.5 & 2.6 which are still listed/supported here, there's a couple examples of how to do it with ruby 2.7/3.0 and Rails 5.2 below.